prj_shun/List_of_Contact_Persons_v2.py

```diff
@@ -0,0 +1,200 @@
+import openpyxl
+import numpy as np
+import pandas as pd
+
+PATH = "/Users/shun/Desktop/nCoV_survey/nCoV_survey2004020_もっと修正案200504-2.xlsx"
+
```

Comment:
This should be changed. 
```suggestion

PATH = "./nCoV_survey2004020_もっと修正案200504-2.xlsx"

```